root_locals.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   environment = lower(terraform.workspace)
-  assume_role = local.environment == "mgmt" || local.environment == "sbox" ? "arn:aws:iam::${var.tdr_account_number}:role/IAM_Admin_Role" : "arn:aws:iam::${var.tdr_account_number}:role/TDRTerraformRole${title(local.environment)}"
+  assume_role = var.project == "dr2" ? "arn:aws:iam::${var.account_number}:role/${title(local.environment)}TerraformRole" : local.environment == "mgmt" || local.environment == "sbox" || local.environment == "ddri" ? "arn:aws:iam::${var.account_number}:role/IAM_Admin_Role" : "arn:aws:iam::${var.account_number}:role/TDRTerraformRole${title(local.environment)}"
```

Comment:
There is a hardcoded, `TDRTerraformRole...` should this be made project neutral to account for dr2 IAM roles?

Reply:
I've added the terraform account roles into `da-terraform-configurations` here https://github.com/nationalarchives/da-terraform-configurations/pull/3 and then used this in the assume_role which gets rid of this messy if else if else.